tensorflow/contrib/cudnn_rnn/python/ops/cudnn_rnn_ops.py

```diff
@@ -1027,6 +1028,14 @@ def _cudnn_rnn(inputs,
     args["sequence_lengths"] = sequence_lengths
     args["time_major"] = time_major
     outputs, output_h, output_c, _, _ = gen_cudnn_rnn_ops.cudnn_rnnv3(**args)
+  elif time_major is False:
+    max_time = inputs.get_shape().as_list()[1]
```

Comment:
This only works if the inputs is of static shape (shape known at graph-construction time).

To make it work for a dynamic shaped inputs (and please test w/ unittest. You can try a `tf.placeholder(shape=[None, None, None])` and run a session with feed_dict), try something like the following:
```py
batch_size, max_len = tf.get_shape(inputs)[0], tf.get_shape(inputs)[1]
seq_lengths = tf.fill([batch_size], max_len)
```